### ISSUE TITLE
Broken brew tap version for danger-swift

### ISSUE THREAD

**Issue by Sega-Zero** [danger/homebrew-tap#11]
the commit https://github.com/danger/homebrew-tap/commit/41981bb9b3b9abcdddf2fb1cb9227d4a7f71f45f added a regression, so brew no longer able to install this tap:

```shell
 % brew install danger/tap/danger-swift
Error: invalid attribute for formula 'danger/tap/danger-swift': version (nil)
```

**Comment by ghraibeh** [comment]
I'm experiencing the same problem.

**Comment by Sega-Zero** [comment]
@f-meloni something should be done again:
```bash
% brew install danger/tap/danger-swift                                                                                               
==> Fetching danger/tap/danger-swift
==> Downloading https://github.com/danger/danger-swift/archive/3.17.0.tar.gz
==> Downloading from https://codeload.github.com/danger/swift/tar.gz/refs/tags/3.17.0
  #=O=#     #       #                                                                                                                                                            
Error: danger-swift: SHA256 mismatch
Expected: e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855
  Actual: 39963124f4256c055bb1269f3b58fd23eddcafb1db4968431711abf82d833416
    File: /Users/admin/Library/Caches/Homebrew/downloads/e7391e74b6eec7e1637f4be90e360bdb46f6c66c5161c5c95220c34c3425fdad--swift-3.17.0.tar.gz
To retry an incomplete download, remove the file above.
```

**Comment by f-meloni** [comment]
I have updated the sha with the one in your log, let's see if it works now

**Comment by f-meloni** [comment]
To make things easier I have released 3.17.1, which should be fine now
